osgar/tools/lidarview.py

```diff
@@ -480,7 +480,10 @@ def lidarview(gen, caption_filename, callback=False, out_video=None, jump=None):
                     frames_step = 90
                     sleep_time = 10
                 if event.key == K_s:
-                    pygame.image.save(image, "saveX-{:04}.jpg".format(save_counter))
+                    if use_image2:
+                        pygame.image.save(image2, "saveX-{:04}.jpg".format(save_counter))
+                    else:
+                        pygame.image.save(image, "saveX-{:04}.jpg".format(save_counter))
```

Comment:
What about something like this?
```suggestion
                    save_image = image2 if use_image2 else image
                    pygame.image.save(save_image, "saveX-{:04}.jpg".format(save_counter))
```
The core of the suggestion is not to duplicate the whole save call.